packages/gitlab/src/components/GitlabCI/GitlabCI.tsx

```diff
@@ -9,12 +9,16 @@ import {
     MergeRequestStats,
     PipelinesTable,
     ReleasesCard,
+    ReadmeCard,
 } from '../widgets';
 
 export const GitlabCI = () => (
     <Page themeId="tool">
         <Content>
             <Grid container spacing={6} direction="row" alignItems="stretch">
+                <Grid item md={12}>
+                    <ReadmeCard />
```

Comment:
Would it make sense to opt-in on this feature through the `app-config.yaml`?

For example
```yaml
gitlab:
    defaultCodeOwnersPath: .gitlab/CODEOWNERS
    proxyPath: /gitlabci
    readme:
        enabled: true
        defaultPath: README.md
```

Reply:
I think we can add it to the `GitlabCI` React component. By default should be disabled.

Reply:
Since the only way to pass props to the `GitlabCI` is through the entity provider or plugin configuration. I'm removing the ReadmeCard.
I'm adding it to point 3 on the README under the (**Optional**) by using the `EntityGitlabReadmeCard`

Reply:
Ok LGTM !